lib/bookmark_reminder_notification_handler.rb

```diff
@@ -9,13 +9,21 @@ def self.send_notification(bookmark)
       end
       return unless bookmark.topic
 
-      create_notification(bookmark)
+      begin
+        create_notification(bookmark)
+      rescue => err
+        Rails.logger.warn("Bookmark #{bookmark.id} failed to send notification with error: #{err}\n#{err.backtrace.join("\n")}")
+      end
 
```

Comment:
still not a fan of eating this error. the method says send notification, so it really should do so.

callers can then decide how they want to deal with the outlier conditions.

as it stands this would bubble all the way up and be visible in sidekiq logs.

Reply:
I suppose the error must be with the update then, because there are no errors in the sidekiq logs currently. I reverted the log changes and this PR will just change update to `update!`, next time the error should bubble